prober/file.go

```diff
@@ -5,12 +5,13 @@ import (
 	"fmt"
 
 	"github.com/bmatcuk/doublestar/v2"
+	"github.com/go-kit/log"
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/ribbybibby/ssl_exporter/config"
 )
 
 // ProbeFile collects certificate metrics from local files
-func ProbeFile(ctx context.Context, target string, module config.Module, registry *prometheus.Registry) error {
+func ProbeFile(logger log.Logger, ctx context.Context, target string, module config.Module, registry *prometheus.Registry) error {
```

Comment:
The context should be the first argument, as suggested in https://golang.org/pkg/context/.

>  Do not store Contexts inside a struct type; instead, pass a Context explicitly to each function that needs it. The Context should be the first parameter, typically named ctx:
>
> ```
> func DoSomething(ctx context.Context, arg Arg) error {
> 	// ... use ctx ...
> }
> ```